lib/semantics/resolve-names.cc

```diff
@@ -2790,6 +2791,22 @@ Symbol &SubprogramVisitor::PushSubprogramScope(
   return *symbol;
 }
 
+Symbol &SubprogramVisitor::PushBlockDataScope(const parser::Name &name) {
```

Comment:
This function is only called from one place that doesn't use its return value.  Should it just return void?

Reply:
Sure.